src/vm/ast/definition_sorter/mod.rs

```diff
@@ -94,6 +94,7 @@ impl <'a> DefinitionSorter {
             },
             List(ref exprs) => {
                 // Avoid looking for dependencies in tuples
+                // TODO: Eliminate special handling of tuples as it is a separate presymbolic expression type
```

Comment:
Is this todo resolved?

Reply:
No -- this PR leaves the "implicit" tuple syntax intact -- I think we should plan to open a subsequent PR which:

1. Removes the implicit tuple syntax.
2. Allows data-maps to store keys and values of non-tuple types (basically like the data-vars)

That will allow us to remove all of these special case code for tuples, and make a lot of the syntax more consistent (plus, simplify any contract that wants to use tuples with just single values in data-maps).

Reply:
See also the earlier comment about obsoleting the [get_definition_type_of_tuple_argument](https://github.com/blockstack/blockstack-core/blob/719b539ba7f5c084ca6f73d3398f2f22c0a2563c/src/vm/functions/tuples.rs#L65) function.